cmd/minishift/cmd/oc_env.go

```diff
@@ -42,12 +44,19 @@ type OcShellConfig struct {
 }
 
 func getOcShellConfig(ocPath, forcedShell string) (*OcShellConfig, error) {
+	var cmdLine string
+
 	userShell, err := shell.GetShell(forcedShell)
 	if err != nil {
 		return nil, err
 	}
 
-	cmdLine := "minishift oc-env"
+	if constants.IsProfileExists() {
```

Comment:
_constants.IsProfileExists()_ - a couple of things. contents.go is again not a good place to add this functionality. Maybe profile related functions can be collected in a new package/file. Also, imo the function should be called _ProfileExists_ or _DoesProfileExists_. IMO common practice is the former.